doc/configtables/electricity.csv

```diff
@@ -5,7 +5,9 @@ co2limit,:math:`t_{CO_2-eq}/a`, float, "Cap on system total annual carbon dioxid
 co2base,:math:`t_{CO_2-eq}/a`, float, "Reference value of system total annual carbon dioxide equivalent emissions. Used only if relative emission reduction target is specified in ``{opts}`` wildcard."
 automatic_emission, bool, "{True, False}", "True: Emissions are obtained from automatic emission extraction procedure. False: Emissions are obtained manually"
 automatic_emission_base_year,, integer, "CO2 emissions of year 1990 from EDGAR category 1A1a (Public electricity and heat production)."
-agg_p_nom_limits, file, path, "Reference to ``.csv`` file specifying per carrier generator nominal capacity constraints for individual countries if ``'CCL'`` is in ``{opts}`` wildcard. Default is ``data/agg_p_nom_minmax.csv``."
+agg_p_nom_limits,,, "Configure per carrier generator nominal capacity constraints for individual countries if ``'CCL'`` is in ``{opts}`` wildcard."
```

Comment:
I wonder if we can improve a bit clarity of this fragment keeping, also for first-time users who is trying to understand the essence of the constraint? Sketching a suggestion bellow; feel free to revise obviously

```suggestion
agg_p_nom_limits,,, "Configure per carrier generator capacity constraints limiting minimum and maximum values of the expanded capacity `p_nom_opt` for individual countries. Is enabled if ``'CCL'`` is presents in ``{opts}`` wildcard."
```